content/docs/get-started/index.md

```diff
@@ -165,3 +162,363 @@ requirements:
+
+Load the model we saved earlier in a simple Python script to predict some
+probabilities.
+
+Create this `predict.py` script:
+
+```py
+from mlem.api import load
+
+model = load("models/rf")  # RandomForestClassifier
+features = [
+    "sepal length (cm)",
+    "sepal width (cm)",
+    "petal length (cm)",
+    "petal width (cm)",
+]
+x = pd.DataFrame([[0, 1, 2, 3]], columns=features)
+y_pred = model.predict_proba(x)
+
+print(y_pred)
+```
+
+Now, let's run the script:
+
+```cli
+$ python predict.py
+[[0.47 0.24 0.29]]
+```
+
+We see that the prediction probabilities were successfully printed to stdout.
+
+</tab>
+
+<tab title="Command Line">
+
+### Batch scoring
+
+The MLEM CLI allows us to natively use any saved model directly for prediction
+or batch scoring with any local dataset. This is very handy if we want to get
+some quick feedback about a model we just created.
+
+First, create an example dataset file to apply, we'll go with a `csv` format:
+
+```cli
+$ echo "sepal length (cm),sepal width (cm),petal length (cm),petal width (cm)
+0,1,2,3" > new_data.csv
+```
+
+Next, simply run `mlem apply` to apply this dataset against our model's
+`predict_proba` method:
+
+```cli
+$ mlem apply models/rf new_data.csv \
+    --method predict_proba \
+    --import \
+    --import-type "pandas[csv]"
+⏳️ Importing object from new_data.csv
+⏳️ Loading model from models/rf.mlem
+🍏 Applying `predict_proba` method...
+[[0.47 0.24 0.29]]
+```
+
+And we get our expected prediction probabilities as output.
+
+<details>
+
+#### Learn more about the CLI options used
+
+- The `--method`/`-m` flag tells MLEM to invoke the `predict_proba` method and
+  return the class probabilities, instead of the default `predict`.
+- The `--import`/`-i` flag tells MLEM to import the data on the fly.
+- The `--import-type` / `--it` flag, helps MLEM understand the data format.
+  Here, it's `pandas[csv]` - a csv file that should be read with Pandas. For
+  that to work, your data should be in a format that is supported by
+  [MLEM import](/doc/user-guide/importing). You can learn more about specifying
+  these arguments on `mlem apply` page.
+
+Alternatively, you could save the dataset itself
+[using MLEM](/doc/user-guide/data) to use `mlem apply` on it.
+
+</details>
+
+</tab>
+</toggle>
+
+We saw that MLEM provides a consistent and friendly way for you to work with
+models both via API and CLI. However, MLEM **really** shines when you need to
+package and deploy your models, either as part of an application or locally for
+testing.
+
+## Deploying and Serving models
+
+MLEM can serve a model for you using different server implementations, for
+example FastAPI or RabbitMQ. Here we'll check out how it works with FastAPI
+since serving models via a REST API is a very common use case.
+
+## Local model serving
+
+First thing first, let's run a model server locally on our machine. To launch a
+local FastAPI model server, simply run:
+
+```cli
+$ mlem serve fastapi --model models/rf
```

Comment:
- [ ] I thought we had decided to leave this out of the GS? Focus on deployment.

Reply:
I'm not aware of this - it depends how aggressively we want to cut length here. @aguschin  - thoughts?

Reply:
I bet almost every user would run `mlem serve` multiple times for his model until he decides to do `mlem deploy`. So it's good that we introduce it here.

Also, off topic - maybe we need to `mlem serve streamlit` in GS? It will have a nicer look I assume.

Reply:
Sounds good, we should keep it then.

> maybe we need to mlem serve streamlit

Up to you.

Reply:
I think fastapi works better for us - better adoption and swagger api spec are a great place to showcase a "by the book" RESTapi server